### PR TITLE
Update projects.yml

### PR DESCRIPTION
**Pull request by jtreuting**
Added SharpRepository

### CODE REVIEW

File: _data/projects.yml [hunk 1]
```diff
@@ -181,3 +181,11 @@
   upforgrabs:
     name: up-for-grabs
     link: https://github.com/JakeGinnivan/GitReleaseNotes/issues?labels=up-for-grabs&state=open
+
+- name: SharpRepository
+  desc: Generic Repository with implementations for EF, RavenDb, MongoDB, Db4o and more
+  site: https://github.com/SharpRepository/SharpRepository
+  tags: [Repository, C#, EntityFramework]]
```
Comment (dahlbyk) [inline _data/projects.yml]: Extra ] here

Comment (jtreuting) [inline _data/projects.yml]: sorry about that, fixed now

Comment (dahlbyk) [inline _data/projects.yml]: No worries, that's what code review is for

**Comment by dahlbyk** [comment]
Thanks - neat project!

**Comment by jtreuting** [comment]
Thanks, and thank for including in the list.  It's been great to check out other projects that I had never seen before.  Will hopefully get involved in a couple of them.

**Pull request merged**